lepton/starterkits/non-english-llm-reasoning.ipynb

```diff
@@ -0,0 +1,1035 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Training a non-English Reasoning Language Model on Lepton with NeMo 2.0\n",
+        "\n",
+        "[NVIDIA NeMo Framework](https://docs.nvidia.com/nemo-framework/user-guide/latest/overview.html) is a powerful toolkit for training and fine-tuning Large Language Models. This notebook demonstrates how to create a reasoning-capable LLM for Spanish on Lepton using NeMo 2.0.\n",
+        "\n",
+        "While many state-of-the-art models perform well in English, their performance often degrades significantly in other languages. This tutorial shows how data translation, continued pre-training (CPT), and supervised fine-tuning (SFT) can be combined to create a model that reasons in English and outputs answers in Spanish. This approach maintains strong reasoning capabilities while enabling Spanish language output for better accessibility.\n",
+        "\n",
+        "## Objectives\n",
```

Comment:
For more visual differentiation on the platform, can you update this header to level 3 (`###`)?